packages/launchpad/src/migration/OptOutModal.cy.tsx

```diff
@@ -0,0 +1,7 @@
+import OptOutModal from './OptOutModal.vue'
+
+describe('<OptOutModal/>', { viewportWidth: 1119 }, () => {
+  it('renders expected content', () => {
+    cy.mount(OptOutModal)
+  })
+})
```

Comment:
Let's add a couple more component tests:
* with `hasCustomIntegrationFolder` enabled
* with `skip` or `renameFolder` selected so we can assert the additional text

We could take percy snapshots for all as well.